static/js/jira-configuration.js

```diff
@@ -74,15 +76,44 @@ $('.sync-connection-link').click(function (event) {
 			}
 		});
 	});
-
 })
 
+$(".sync-connection-link").click(function (event) {
+	event.preventDefault();
+	const installationId = $(event.target).data("installation-id");
+	const jiraHost = $(event.target).data("jira-host");
+	const csrfToken = document.getElementById("_csrf").value;
+
+	window.AP.context.getToken(function (token) {
+		$.ajax({
+			type: "POST",
+			url: `/jira/sync`,
+			data: {
+				installationId,
+				jiraHost,
+				syncType: "full",
+				jwt: token,
+				_csrf: csrfToken,
+			},
+			success: function (data) {
+				AP.navigator.reload();
+			},
+			error: function (error) {
+				console.log(error);
+			},
+		});
+	});
+});
+
+/* ***************************** */
+/* To be removed after FF tested */
```

Comment:
More code I would've handled differently if I'd planned to (or knew how to) use FF flags before I did this work. Will be cleaned up after these changes are deployed and thoroughly tested.